packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1320,6 +1323,11 @@ module.exports = function (log, config, bounces, statsd) {
         supportLinkAttributes: links.supportLinkAttributes,
         supportUrl: links.supportUrl,
         time,
+        // Only show the banner warning for new device AMO logins, or web
+        // logins with no service associated
+        showBannerWarning:
+          message.service === undefined ||
```

Comment:
Sync (including desktop v3) should send up `service` as `'sync'` and then our other RPs have `client_id`s associated.